src/components/Features.tsx

```diff
@@ -0,0 +1,50 @@
+import { LINKS } from '@constants';
+import { HeartIcon, OpenSourceIcon, SolidModelingIcon } from '@icons';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { FeatureBlock } from './common/FeatureBlock';
+
+interface IProps { }
+
+export const Features: React.FC<IProps> = () => {
+  const { t } = useTranslation();
+  const features = [
+    {
+      index: 1,
+      Icon: OpenSourceIcon,
+      title: t('FEATURES.FEATURE_1.TITLE'),
+      description: t('FEATURES.FEATURE_1.DESCRIPTION'),
+    },
+    {
+      index: 2,
+      Icon: HeartIcon,
+      title: t('FEATURES.FEATURE_2.TITLE'),
+      description: t('FEATURES.FEATURE_2.DESCRIPTION'),
+      linkName: t('FEATURES.FEATURE_2.BUTTON'),
+      link: LINKS.LEARN_MORE_PDF,
+    },
+    {
+      index: 3,
+      Icon: SolidModelingIcon,
+      title: t('FEATURES.FEATURE_3.TITLE'),
+      description: t('FEATURES.FEATURE_3.DESCRIPTION'),
+    },
+  ];
+  return (
+    <section className="features">
```

Comment:
There are some places where everything is joined line to line. If it's possible and to make code breath a little can we add a new line after the function ends, return starts, prop types start, etc. You can make a decision where to add new line, it will make code a little more cleaner

Reply:
@divyanshugarg36 just a reminder, not an issue.

Reply:
Oh yeah,
This one has been taken care of
Each component have required spacings